src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.h

```diff
@@ -1,17 +1,29 @@
 #ifndef Z_OBJ_TOKEI_TOBIRA_H
 #define Z_OBJ_TOKEI_TOBIRA_H
 
+#define OBJTOKEITOBIRA_GET_TYPE(thisx) ((thisx)->params & 1)
+#define OBJTOKEITOBIRA_PARAMS(type) (type)
+
 #include "global.h"
```

Comment:
```suggestion
#include "global.h"

#define OBJTOKEITOBIRA_GET_TYPE(thisx) ((thisx)->params & 1)
#define OBJTOKEITOBIRA_PARAMS(type) (type)
```
We typically have includes at the top of the file, so the params can be moved down